pkg/api/admin/openshiftcluster_convert.go

```diff
@@ -193,6 +193,7 @@ func (c *openShiftClusterConverter) ToInternal(_oc interface{}, out *api.OpenShi
 	out.Properties.NetworkProfile.PodCIDR = oc.Properties.NetworkProfile.PodCIDR
 	out.Properties.NetworkProfile.ServiceCIDR = oc.Properties.NetworkProfile.ServiceCIDR
 	out.Properties.NetworkProfile.MTUSize = api.MTUSize(oc.Properties.NetworkProfile.MTUSize)
+	out.Properties.NetworkProfile.UserDefinedRouting = api.UserDefinedRouting(oc.Properties.NetworkProfile.UserDefinedRouting)
```

Comment:
Need to add a conversion in `toExternal` as well so we can view it from an admin API perspective.

Reply:
+1